cmd/crypto/help.go

```diff
@@ -38,29 +38,64 @@ var (
 		},
 		config.HelpKV{
 			Key:         KMSVaultAppRoleID,
-			Description: `unique role ID for approle`,
+			Description: `AppRole Role ID used to authenticate to Vault`,
 			Type:        "string",
 		},
 		config.HelpKV{
 			Key:         KMSVaultAppRoleSecret,
-			Description: `unique secret ID for approle`,
+			Description: `AppRole Secret ID used to authenticate to Vault`,
 			Type:        "string",
 		},
 		config.HelpKV{
 			Key:         KMSVaultNamespace,
-			Description: `optional KMS namespace e.g. "customer1"`,
+			Description: `optional Vault namespace - e.g. "customer1"`,
 			Optional:    true,
 			Type:        "string",
 		},
 		config.HelpKV{
 			Key:         KMSVaultKeyVersion,
-			Description: `optional key version number`,
+			Description: `optional key version if not the latest should be used`,
 			Optional:    true,
 			Type:        "number",
 		},
 		config.HelpKV{
 			Key:         KMSVaultCAPath,
-			Description: `optional path to PEM-encoded CA certs e.g. "/home/user/custom-certs"`,
+			Description: `path to PEM-encoded CA certs to verify Vault's TLS certificate - e.g. "/home/user/custom-root-CAs"`,
+			Optional:    true,
+			Type:        "path",
+		},
+		config.HelpKV{
+			Key:         config.Comment,
+			Description: config.DefaultComment,
+			Optional:    true,
+			Type:        "sentence",
+		},
+	}
+
+	HelpKeys = config.HelpKVS{
+		config.HelpKV{
+			Key:         KMSKeysEndpoint,
+			Description: `API endpoint - e.g. "https://key-endpoint:7373"`,
+			Type:        "url",
+		},
+		config.HelpKV{
+			Key:         KMSKeysKeyName,
+			Description: `key at the key server used to encrypt objects - e.g. "my-minio-key"`,
+			Type:        "string",
+		},
+		config.HelpKV{
+			Key:         KMSKeysCertFile,
+			Description: `MinIO's client certificate for TLS authentication - e.g. /etc/keys/public.crt`,
+			Type:        "path",
+		},
+		config.HelpKV{
+			Key:         KMSKeysKeyFile,
+			Description: `MinIO's private key for TLS authentication e.g. /etc/keys/private.key`,
+			Type:        "path",
+		},
+		config.HelpKV{
+			Key:         KMSKeysCAPath,
+			Description: `Path to PEM-encoded certificate(s) to verify the TLS certificate of the key server - e.g. /etc/keys/CAs`,
```

Comment:
This should be shorter like `optional path to PEM-encoded CA certs e.g. "/etc/keys/CAs"`